docs/docsite/rst/network/index.rst

```diff
@@ -4,36 +4,14 @@
 Ansible for Network Automation
 ******************************
 
-Introduction
-============
-
 Ansible Network modules extend the benefits of simple, powerful, agentless automation to network administrators and teams. Ansible Network modules can configure your network stack, test and validate existing network state, and discover and correct network configuration drift.
 
-If you're new to Ansible, or new to using Ansible for network management, start with the Getting Started Guide.
+If you're new to Ansible, or new to using Ansible for network management, start with the :doc:`Getting Started Guide<getting_started/index>` If you are already familiar with network automation with Ansible, see :doc:`Advanced Topics with Ansible for  Network Automation<user_guide/index>`.
```

Comment:
Please use :ref: instead of :doc: for all internal references, and add anchors as needed, so we can move content around later (:doc: links redirect based on filenames, while :ref: links redirect based on anchors).